@AH-64D Apache Longbow/Addons/fza_ah64_ufd/functions/fn_update.sqf

```diff
@@ -77,9 +77,9 @@ do {
 
         for "_i" from 0 to 5 do {
             _heli setUserMFDText [MFD_TEXT_IND_UFDTEXT0 + _i, format["%1|%2|%3",
-                if (count _warnings > _i) then {_warnings # _i} else {"          "},       //10
```

Comment:
Won't all advisories need resizing to do this?

If so that seems like a large alternate PR?

Reply:
Shouldn't, this defines the padding IIRC

Reply:
![image](https://github.com/AH-64D-Apache-Official-Project/AH-64D/assets/9770674/7c553d1b-addf-4715-9363-eba3222d556f)

Reply:
If we are resizing these, that should be a different PR